test/Unit/Porter/Net/Caching/CachingConnectorTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace ScriptFUSIONTest\Unit\Porter\Net\Caching;
+
+use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
+use ScriptFUSION\Porter\Cache\MemoryCache;
+
+final class CachingConnectorTest extends \PHPUnit_Framework_TestCase
+{
+    use MockeryPHPUnitIntegration;
+
+    public function test()
```

Comment:
* I would expect this test case to be added to the existing [CachingConnectorTest](https://github.com/ScriptFUSION/Porter/blob/master/test/Integration/Porter/Connector/CachingConnectorTest.php)
* This test case should have a name, e.g. *testHashExcludesBannedCharacters*.